pom.xml

```diff
@@ -215,12 +216,12 @@
                     </configuration>
                     <executions>
                         <execution>
-                            <phase>compile</phase>
+                            <phase>clean</phase>
```

Comment:
shouldn't this be `validate`? keep in mind clean belongs to a different lifecycle in maven